pyopensprinkler/__init__.py

```diff
@@ -116,6 +117,12 @@ def _refresh_state(self):
         (_, content) = self.request("/ja")
         self._state = content
 
+        fpr0 = self._get_option("fpr0")
```

Comment:
I would say we leave these in the `refresh` method to keep this method strictly scoped at accessing the API and storing the response.

Reply:
Addressed.